firmware_configurations/klipper/RealDeuce/MKS-Makerbase/Monster8_v1.0_003/Voron2_Monster8_Config.cfg

```diff
@@ -0,0 +1,531 @@
+## Voron Design VORON2.4 250/300/350mm Monster8 TMC2209 UART config
+
+## *** THINGS TO CHANGE/CHECK: ***
+## MCU paths                        [mcu] section
+## Thermistor types                 [extruder] and [heater_bed] sections - See 'sensor types' list at end of file
+## Z Endstop Switch location        [safe_z_home] section
+## Homing end position              [gcode_macro G32] section
+## Z Endstop Switch  offset for Z0  [stepper_z] section
+## Probe points                     [quad_gantry_level] section
+## Min & Max gantry corner postions [quad_gantry_level] section
+## PID tune                         [extruder] and [heater_bed] sections
+## Fine tune E steps                [extruder] section
+## Lines beginning with #! describe where to connect wires.
+## Lines beginning with ##----- are placed around sections that need
+##  to be customized for your build size or other hardware
+
+[mcu]
+## Obtain definition by "ls -l /dev/serial/by-id/"
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/usb-Klipper_stm32f407xx_42002B000151373431373036-if00
```

Comment:
just to avoid confusion this should be something like `serial /dev/serial/by-id/{YOUR SERIAL}` so the user is forced to populate it.